shopfloor/actions/data.py

```diff
@@ -49,6 +49,7 @@ def _picking_parser(self):
             "bulk_line_count",
             "total_weight:weight",
             "scheduled_date",
+            "progress",
```

Comment:
Are we sure we need to load this information for ALL the pickings and for all the cases?
AFAIS on https://github.com/OCA/stock-logistics-workflow/pull/1116 it can be heavy (unless you improve perf)

Reply:
For the moment the performance is good. 

Should I create a different parser just for this?

Reply:
as we said, pls make it optional